OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -19,15 +19,71 @@ class CheckSequences : ILintPass
 	{
 		public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 		{
-			var sequences = MiniYaml.MergeLiberal(map.SequenceDefinitions, Game.ModData.Manifest.Sequences.Select(s => MiniYaml.FromFile(s)).Aggregate(MiniYaml.MergeLiberal));
+			var sequenceDefinitions = MiniYaml.MergeLiberal(map.SequenceDefinitions,
+				Game.ModData.Manifest.Sequences.Select(s => MiniYaml.FromFile(s)).Aggregate(MiniYaml.MergeLiberal));
 
 			foreach (var actorInfo in map.Rules.Actors)
+			{
 				foreach (var renderInfo in actorInfo.Value.Traits.WithInterface<RenderSimpleInfo>())
```

Comment:
This request is so complicated in itself and essentially unrelated that I moved it into pull request https://github.com/OpenRA/OpenRA/pull/8296.